lib/JSON/Validator/Util.pm

```diff
@@ -18,7 +17,11 @@ our @EXPORT_OK
 sub E { JSON::Validator::Error->new(@_) }
 
 sub data_checksum {
-  Mojo::Util::md5_sum(Data::Dumper->new([@_])->Sortkeys(1)->Useqq(1)->Dump);
+  return Mojo::Util::md5_sum(
+    ref $_[0]
+    ? Mojo::JSON::encode_json([@_])
```

Comment:
I don't trust this. I used Data::Dumper for a reason, I cannot remember which case failed, but something has failed before. It's a bit annoying, since it seems like that code was in Swagger2::SchemaValidator, which is history 😥

Reply:
Ok. Now using Sereal::Encoder if available and YAML::XS if not.